### ISSUE TITLE
Breaks build in cli 2.13

### ISSUE THREAD

**Issue by vlascik** [fivetanley/ember-promise-helpers#20]
This addon breaks the build in 2.13 cli, presumably because of a switch to Babel 6.
```
File: project/helpers/await.js
The Broccoli Plugin: [broccoli-persistent-filter:Babel] failed with:
SyntaxError: project/helpers/await.js: await is a reserved word (1:18)
> 1 | export { default, await } from 'ember-promise-helpers/helpers/await';
    |                   ^
  2 | 
    at Parser.pp$5.raise (P:\Project\node_modules\babylon\lib\index.js:4373:13)
    at Parser.pp$3.checkReservedWord (P:\Project\node_modules\babylon\lib\index.js:4266:10)
    at Parser.pp$3.parseIdentifier (P:\Project\node_modules\babylon\lib\index.js:4243:10)
    at Parser.pp$1.parseExportSpecifiers (P:\Project\node_modules\babylon\lib\index.js:2779:23)
    at Parser.pp$1.parseExport (P:\Project\node_modules\babylon\lib\index.js:2593:28)
    at Parser.pp$1.parseBlockBody (P:\Project\node_modules\babylon\lib\index.js:2223:21)
    at Parser.pp$1.parseTopLevel (P:\Project\node_modules\babylon\lib\index.js:1733:8)
    at File.parse (P:\Project\node_modules\ember-cli-babel\node_modules\babel-core\lib\transformation\file\index.js:517:15)
    at File.parseCode (P:\Project\node_modules\ember-cli-babel\node_modules\babel-core\lib\transformation\file\index.js:602:20)
    at P:\Project\node_modules\ember-cli-babel\node_modules\babel-core\lib\transformation\pipeline.js:49:12

The broccoli plugin was instantiated at: 
    at Babel.Plugin (P:\Project\node_modules\broccoli-plugin\index.js:7:31)
    at Babel.Filter [as constructor] (P:\Project\node_modules\broccoli-persistent-filter\index.js:62:10)
    at new Babel (P:\Project\node_modules\ember-cli-babel\node_modules\broccoli-babel-transpiler\index.js:35:10)
    at Babel (P:\Project\node_modules\ember-cli-babel\node_modules\broccoli-babel-transpiler\index.js:30:12)
    at P:\Project\node_modules\ember-cli-preprocess-registry\preprocessors.js:180:26
    at Array.forEach (native)
    at processPlugins (P:\Project\node_modules\ember-cli-preprocess-registry\preprocessors.js:178:11)
    at EmberApp.appAndDependencies (P:\Project\node_modules\ember-cli\lib\broccoli\ember-app.js:1101:27)
    at EmberApp.javascript (P:\Project\node_modules\ember-cli\lib\broccoli\ember-app.js:1218:30)
    at EmberApp.toArray (P:\Project\node_modules\ember-cli\lib\broccoli\ember-app.js:1655:12)
```

**Comment by tylerturdenpants** [comment]
@vlascik you can try my PR #19 and see if that resolves your issue.

**Comment by vlascik** [comment]
@tylerturdenpants yep, that did it, thanks

**Comment by tylerturdenpants** [comment]
You're welcome! @vlascik I'm not sure how to get @fivetanley attention, but I'm glad it works for someone other than me.

**Comment by fivetanley** [comment]
This was fixed by #19 i think
